spec/features/admin_user/admin_user_home_page_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.